apps/configurator/lib/configurator_web/controllers/utils_api.ex

```diff
@@ -0,0 +1,25 @@
+defmodule ConfiguratorWeb.UtilsAPI do
```

Comment:
I believe just Utils is a better name than UtilsAPI.

Reply:
I agree!

Reply:
Done